src/WebCompiler/Compile/SassCompiler.cs

```diff
@@ -117,42 +114,38 @@ private void RunCompilerProcess(Config config, FileInfo info)
                 p.WaitForExit();
 
                 _output = stdout.Result;
-                // postcss outputs "√ Finished stdin (##ms)" to stderr for some reason
-                if (!stderr.Result.StartsWith("√"))
+                if (!string.IsNullOrEmpty(stderr.Result))
+                {
                     _error = stderr.Result;
+                }
             }
         }
 
         private static string ConstructArguments(Config config)
         {
-            string arguments = "";
+            var arguments = new StringBuilder();
 
             SassOptions options = SassOptions.FromConfig(config);
 
             if (options.SourceMap || config.SourceMap)
-                arguments += " --source-map-embed=true";
-
-            arguments += " --precision=" + options.Precision;
+                arguments.Append(" --embed-source-map");
 
-            if (!string.IsNullOrEmpty(options.OutputStyle))
-                arguments += " --output-style=" + options.OutputStyle;
+            if (options.Quiet)
+                arguments.Append(" --quiet");
 
-            if (!string.IsNullOrEmpty(options.IndentType))
-                arguments += " --indent-type=" + options.IndentType;
+            if (options.QuietDeps)
+                arguments.Append(" --quiet-deps");
 
-            if (options.IndentWidth > -1)
-                arguments += " --indent-width=" + options.IndentWidth;
+            arguments.Append(" --precision=" + options.Precision);
+            arguments.Append(" --style=" + options.Style.ToString().ToLowerInvariant());
 
-            if (!string.IsNullOrEmpty(options.IncludePath))
-                arguments += " --include-path=" + options.IncludePath;
-
-            if (!string.IsNullOrEmpty(options.SourceMapRoot))
-                arguments += " --source-map-root=" + options.SourceMapRoot;
-
-            if (!string.IsNullOrEmpty(options.LineFeed))
-                arguments += " --linefeed=" + options.LineFeed;
```

Comment:
@XzaR90 @failwyn does this PR remove the ability to specify the line feed option? In particular it seems this is causing LF behavior to differ between VS2022 and VS2019 versions

See: https://github.com/failwyn/WebCompiler/commit/8c7d2ce795eb23700bcaeddcd6bc4f457d77286c

Reply:
It's an option from Node Sass that isn't supported by Dart Sass, there are some differences in the output, but I have not been able to identify any breaking changes yet (please let me know if you find any); the main differences are:

1. the closing brace is on a new line
2. some styles build from lists are in a different order
3. some styles don't have line breaks in them where they used to

Node Sass is obsolete and is no longer being updated, so we moved to Dart Sass.

Reply:
The biggest issue I am seeing with it is that files are constantly getting regenerated with different output between devs on VS2022 and VS2019 on any change. This causes a lot of noise in PRs and review and potential merge conflicts. It would be nice if there was a way to opt out of this behavior at least until more people are using VS2022.

Reply:
I’ll take a look tomorrow and see if I can add a switch to use Node or Dart Sass.

Reply:
1.14.1 has been uploaded to the Visual Studio Marketplace, backwards compatibility with Node Sass is available using the following configuration.

```
 {
    "inputFile": "Styles/kendoui/bootstrap/scss/all.scss",
    "outputFile": "wwwroot/css/kendoui.bootstrap.css",
    "useNodeSass": true,
    "options": {
      "sourceMap": true,
      "relativeUrls": false
    }
  }
```

I still need to restore the Node Sass tests when I have time.